libs/public-pages/src/pages/file-publication/index.ts

```diff
@@ -0,0 +1,46 @@
+import { getArtefactById, getUploadedFile, mockListTypes } from "@hmcts/publication";
+import { formatDateAndLocale } from "@hmcts/web-core";
+import type { Request, Response } from "express";
+import { cy } from "../artefact-not-found/cy.js";
+import { en } from "../artefact-not-found/en.js";
```

Comment:
Probably should not include the language resources directly from another page. If you use redirect for artefact_not_found below, that can be avoided